test/Microsoft.ApplicationInsights.AspNet.Tests/project.json

```diff
@@ -8,10 +8,11 @@
         "Microsoft.AspNet.Mvc.Core": "6.0.0-beta4-*",
         "Microsoft.Framework.Logging": "1.0.0-beta4-*",        
         "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-beta4-*",        
-        "xunit.runner.aspnet": "2.0.0-aspnet-beta4-*"
+        "xunit": "2.1.0-beta2-*",
+        "xunit.runner.dnx": "2.1.0-beta2-*"
```

Comment:
Is this a replacement for `xunit.runner.aspnet`? Does it work in VS 2015 RC as well?